packages/nuxt/src/components/plugins/loader.ts

```diff
@@ -17,10 +17,11 @@
   experimentalComponentIslands?: boolean
 }
 
-const REPLACE_COMPONENT_TO_DIRECT_IMPORT_RE = /(?<=[ (])_?resolveComponent\(\s*["'](lazy-|Lazy(?=[A-Z]))?([^'"]*)["'][^)]*\)/g
+const REPLACE_COMPONENT_TO_DIRECT_IMPORT_RE = /(?<=[ (])_?resolveComponent\(\s*["'](lazy-|Lazy(?=[A-Z]))?(Idle|Visible|idle-|visible-|Event|event-|Media|media-|If|if-|Never|never-|Time|time-|Promise|promise-)?([^'"]*)["'][^)]*\)/g
 export const LoaderPlugin = (options: LoaderOptions) => createUnplugin(() => {
   const exclude = options.transform?.exclude || []
   const include = options.transform?.include || []
+  const clientDelayedComponentRuntime = resolve(distDir, 'components/runtime/client-delayed-component')
```

Comment:
This will be removed in the next push, as it was legacy code from before Nuxt was used the new unplugin vfs. Therefore, I am marking this as resolved now, before making the commit.